scripts/test_runner.sh

```diff
@@ -14,7 +14,7 @@ function validate_test_image() {
 
 function validate_dir(){
   case "$TEST_DIR" in
-       upgradecluster|versionbump|mixedoscluster|dualstack|validatecluster|createcluster|selinux|certrotate|restartservice)
+       upgradecluster|versionbump|mixedoscluster|dualstack|validatecluster|createcluster|selinux|certrotate|secretsencrypt|restartservice)
```

Comment:
Nit: This will keep getting bigger, can we multiline it?

Reply:
fixed